R/helperFunctions.R

```diff
@@ -151,18 +152,17 @@ check_valid_color <- function(color) {
     FALSE
   }
 }
-
-#' @name col2hex
-#' @description convert rgb to hex
-#' @param creator my.col
-#' @keywords internal
+# `col2hex()` ------------------------------------------------------------------
+#' Convert rgb to hex
+#'
+#' @param my.col my.col
 #' @noRd
 col2hex <- function(my.col) {
   rgb(t(col2rgb(my.col)), maxColorValue = 255)
 }
 
 
-## header and footer replacements
+## header and footer replacements -------
```

Comment:
Is the line intentionally shorter than the others?

Reply:
no, I will put it to 80 characters